cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsProcessor.java

```diff
@@ -535,9 +541,9 @@ private boolean enoughTopicPartitionMetrics(Cluster cluster, int brokerId) {
       boolean result = ((double) missingTopics.size() / topicsInBroker.size()) <= MAX_ALLOWED_MISSING_TOPIC_METRIC_PERCENT
```

Comment:
Can we update the JavaDoc of this method to refer to `MAX_ALLOWED_MISSING_TOPIC_METRIC_PERCENT` and `MAX_ALLOWED_MISSING_PARTITION_METRIC_PERCENT` rather than the hardcoded `99%` mentioned in `...collected more than 99% of the topic level and partition level metrics...`?
Can we also make these two static variables a configuration -- with default values of `0.01`?

Reply:
Updated the javadoc. 

Currently it is hard to add these two configs to` KafkaCruiseControlConfig` and let `CruiseControlMetricsProcessor` pick up the  value from `cruisecontrol.properties` because `CruiseControlMetricsProcessor` is a static member in `CruiseControlMetricsReporterSampler`. To do that we need lazy initialization of this static member which I find unnecessary. 

I am thinking of making `KafkaCruiseControlConfig` a singleton and can be used every where. I will include this in a config patch so we can discuss there.